src/components/GoFormGenerator/GoFormGroup.vue

```diff
@@ -0,0 +1,155 @@
+<template>
+  <div :class="getFieldRowClasses">
+    <label
+      v-if="hasLabel"
+      :for="field.id"
+      :class="field.labelClasses"
+      class="control-label m-b-xs"
+    >
+      <span v-html="field.label" />
+      <span
+        v-if='field.help && field.help.label'
+        v-html='field.help.label'
+        class="help-block m-n"
+      />
+    </label>
+    <div>
+      <component
+        v-model="valueInput"
+        :key="field.model"
+        :is="fieldType"
+        :disabled="field.isDisabled"
+        :schema="field"
+        v-validate="field.validations"
+        :data-vv-as="field.label"
+      />
+      <div
+        v-if="field.buttons"
+        class="buttons"
+      >
+        <button
+          v-for="(btn, index) in field.buttons"
+          :key="index"
+          @click="btn.onClick"
+          :class="btn.classes"
+          v-text="btn.label"
+          :type="btn.type || 'button'"
+        />
+      </div>
+    </div>
+    <div
+      class="m-t-xs"
+      v-show="hasErrors || field.hint"
+    >
+      <transition name="fade">
+        <span
+          v-if="hasErrorVeeValidate"
+          class="text-xs text-danger"
+        >
+          {{ errors.first(field.inputName) }}
+        </span>
+        <template
+          v-else-if="hasErrors && !hasErrorVeeValidate"
+          mode="out-in"
+        >
+          <span
+            v-for="(error, index) in fieldModelErrors"
+            :key="index"
+            class="text-xs text-danger"
+          >
+            {{ getModelError(field, error) }}
+          </span>
+        </template>
+      </transition>
+      <div
+        class="text-xs"
+        v-show="field.hint"
+        v-html="field.hint"
+      />
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'GoFormGroup',
+  data () {
+    return {
+      valueInput: this.value,
+      displayFieldsErrors: true,
+      fieldType: `GoField${this.field.type.charAt(0).toUpperCase() + this.field.type.slice(1)}`,
+      isSpecialFieldTypes: ['submit'].includes(this.field.type)
+    }
+  },
+  props: {
+    value: {
+      required: true
+    },
+    options: {
+      type: Object
```

Comment:
If you know the object property we should implement interface to type `options`object.